src/audio/mux/mux.c

```diff
@@ -359,7 +359,7 @@ static int demux_copy(struct comp_dev *dev)
 			continue;
```

Comment:
typo in commit: s/provessing/processing/

Reply:
@akloniex spelling can be fixed in separate PR